README.md

```diff
@@ -8,11 +8,73 @@
 
 Genshin Optimizer is a helper website for the online action-rpg gacha game [Genshin Impact](https://genshin.mihoyo.com/). It is intended to assist players with optimizing artifacts to max-min their characters, while providing a clean, structured UI, and provide real-time results.
 
```

Comment:
Should we add a blurb about where to find the actual hosted site? It's come up a few times where people download the repo and start running `yarn` and such following the readme before they realize the site is already hosted